aptible/database.go

```diff
@@ -184,8 +184,17 @@ func (c *Client) UpdateDatabase(databaseID int64, updates DBUpdates) error {
 }
 
 func (c *Client) DeleteDatabase(databaseID int64) error {
-	params := operations.NewDeleteDatabasesIDParams().WithID(databaseID)
-	_, err := c.Client.Operations.DeleteDatabasesID(params, c.Token)
+	requestType := "deprovision"
+	request := models.AppRequest23{
+		Type: &requestType,
+	}
+	provisionParams := operations.NewPostDatabasesDatabaseIDOperationsParams().WithDatabaseID(databaseID).WithAppRequest(&request)
+	op, err := c.Client.Operations.PostDatabasesDatabaseIDOperations(provisionParams, c.Token)
```

Comment:
```suggestion
	deprovisionParams := operations.NewPostDatabasesDatabaseIDOperationsParams().WithDatabaseID(databaseID).WithAppRequest(&request)
	op, err := c.Client.Operations.PostDatabasesDatabaseIDOperations(deprovisionParams, c.Token)
```

just to clarify the name, take it or leave it